server/api/region.go

```diff
@@ -308,17 +308,17 @@ func (h *regionsHandler) GetAll(w http.ResponseWriter, r *http.Request) {
 }
 
 // @Tags region
-// @Summary List regions start from a key.
-// @Param key query string true "Region start key"
-// @Param endkey query string true "Range end key"
-// @Param limit query integer false "Limit count" default(16)
+// @Summary List regions in a given range[startKey,endKey).
+// @Param startKey query string true "Region range start key"
+// @Param endkey query string true "Region range end key"
+// @Param limit query integer false "Limit count" default(16) without endKey, default(-1) with endKey
```

Comment:
I think it would be better to add a new API to ensure backward compatibility.

Reply:
roger